sentinel-core/src/main/java/com/alibaba/csp/sentinel/node/StatisticNode.java

```diff
@@ -115,10 +116,15 @@ public class StatisticNode implements Node {
     @Override
     public Map<Long, MetricNode> metrics() {
         // The fetch operation is thread-safe under a single-thread scheduler pool.
-        long currentTime = TimeUtil.currentTimeMillis();
-        currentTime = currentTime - currentTime % 1000;
+        final long currentTime = TimeUtil.currentTimeMillis();
+        final long endTime = currentTime - currentTime % 1000;
+        long flushIntervalSec = SentinelConfig.metricLogFlushIntervalSec();
```

Comment:
The `StatisticNode` should not be aware of outside "metric log". Maybe you could refactor `MetricTimerListener` with `StatisticNode#rawMetricsInMin`?